From e2bb4b752d68856c4c307640ae310f47f680aed6 Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Fri, 20 Aug 2021 10:21:07 +0200
Subject: [PATCH 2/4] disable ac97 audio
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
RH-MergeRequest: 39: disable ac97 audio
RH-Commit: [1/1] 5d1bd969d20f960cb0a023f0d6cd1ae5adda22e6 (kraxel/centos-qemu-kvm)
RH-Bugzilla: 1995819
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Philippe Mathieu-Daudé <philmd@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>

RH-Bugzilla: 1995819

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
---
 default-configs/devices/x86_64-rh-devices.mak | 1 -
 1 file changed, 1 deletion(-)

diff --git a/default-configs/devices/x86_64-rh-devices.mak b/default-configs/devices/x86_64-rh-devices.mak
index c2dd112f81..8ae2747bf6 100644
--- a/default-configs/devices/x86_64-rh-devices.mak
+++ b/default-configs/devices/x86_64-rh-devices.mak
@@ -1,6 +1,5 @@
 include rh-virtio.mak
 
-CONFIG_AC97=y
 CONFIG_ACPI=y
 CONFIG_ACPI_PCI=y
 CONFIG_ACPI_CPU_HOTPLUG=y
-- 
2.27.0

